### PR TITLE
Added JPA repository, softdelete, liquibase

### PR DESCRIPTION
**Pull request by ohatsenko**

### CODE REVIEW

**Review by irynamekh — APPROVED** [review]
Good job!

File: pom.xml [hunk 1]
```diff
@@ -59,6 +59,20 @@
...
+            <artifactId>liquibase-core</artifactId>
+            <version>${liquibase.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.liquibase</groupId>
+            <artifactId>liquibase-maven-plugin</artifactId>
+            <version>${liquibase.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
```
Comment (vovasalyha) [inline pom.xml:73]: why do you need this dependency here?

File: src/main/resources/db/changelog/changes/001-create-books-table.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                  name: description
+                  type: varchar(255)
+              - column:
+                  name: cover_image
+                  type: varchar(255)
+              - column:
+                  name: is_deleted
+                  type: boolean
+                  constraints:
+                    nullable: false
```
Comment (vovasalyha) [inline src/main/resources/db/changelog/changes/001-create-books-table.yaml:45]: empty line

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+databaseChangeLog:
+    - include:
+        file: db/changelog/changes/001-create-books-table.yaml
```
Comment (vovasalyha) [inline src/main/resources/db/changelog/db.changelog-master.yaml:3]: empty line

**Review by vovasalyha — APPROVED** [review]
GJ, take a look on few comments

**Pull request merged**